addons/frag/initSettings.inc.sqf

```diff
@@ -3,38 +3,60 @@ private _category = format ["ACE %1", localize LSTRING(Module_DisplayName)];
 [
     QGVAR(enabled), "CHECKBOX",
     [LSTRING(EnableFrag), LSTRING(EnableFrag_Desc)],
-    _category,
+    [_category, LSTRING(Frag)],
     true,
     1
 ] call CBA_fnc_addSetting;
 
 [
     QGVAR(spallEnabled), "CHECKBOX",
     [LSTRING(EnableSpall), LSTRING(EnableSpall_Desc)],
-    _category,
+    [_category, LSTRING(Spall)],
     false,
     1
 ] call CBA_fnc_addSetting;
+
 [
     QGVAR(reflectionsEnabled), "CHECKBOX",
     [LSTRING(EnableReflections), LSTRING(EnableReflections_Desc)],
-    _category,
+    [_category, LSTRING(Reflections)],
     false,
     1
 ] call CBA_fnc_addSetting;
 
 [
-    QGVAR(maxTrack), "SLIDER",
-    [LSTRING(MaxTrack), LSTRING(MaxTrack_Desc)],
-    _category,
-    [0, 50, 10, -1],
+    QGVAR(fragSimComplexity), "LIST",
+     [LSTRING(FragMode), LSTRING(FragMode_Desc)],
+    [_category, LSTRING(Frag)],
+    [[2, 1, 0], [LSTRING(FragMode_Opt2),LSTRING(FragMode_Opt1),LSTRING(FragMode_Opt0)], 2],
     1
 ] call CBA_fnc_addSetting;
 
 [
-    QGVAR(maxTrackPerFrame), "SLIDER",
-    [LSTRING(MaxTrackPerFrame), LSTRING(MaxTrackPerFrame_Desc)],
-    _category,
-    [0, 50, 10, -1],
+    QGVAR(atLeastOne), "CHECKBOX",
+    [LSTRING(MinFrag), LSTRING(MinFrag_Desc)],
+    [_category, LSTRING(Frag)],
+    false,
     1
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(blackList), "EDITBOX",
+    [LSTRING(DefBlackList), LSTRING(DefBlackList_Desc)],
+    [_category, LSTRING(Frag)],
+    QUOTE(['B_556x45_Ball']),
```

Comment:
```suggestion
    '["B_556x45_Ball"]',
```

Reply:
Sounds good, is it a matter of preference or is there another reason I'm missing?

Reply:
Done by https://github.com/acemod/ACE3/pull/9728/commits/13834d7f17bebc2e40cfdc9e192436dffa9d7c4f

Reply:
> Sounds good, is it a matter of preference or is there another reason I'm missing?

Preference.